eth/contracts/StableSwap/StableSwapTwoPool.sol

```diff
@@ -124,58 +139,64 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
     /**
      * @notice initialize
      * @param _coins: Addresses of ERC20 conracts of coins (c-tokens) involved
-     * @param _A: Amplification coefficient multiplied by n * (n - 1)
+     * @param _amplification_coefficient: Amplification coefficient multiplied by n * (n - 1)
      * @param _fee: Fee to charge for exchanges
      * @param _admin_fee: Admin fee
      * @param _owner: Owner
-     * @param _LP: LP address
```

Comment:
So this is a liquidity pool or liquidity provider?

Reply:
liquidity pool